experimental/dds/tree/src/Forest.ts

```diff
@@ -66,6 +66,9 @@ export interface Delta<NodeId> {
 	readonly removed: readonly NodeId[];
 }
 
+/**
+ * @alpha
```

Comment:
This isn't exported; does it need a tag?

Reply:
Yeah, this looks like a mistake.  Nice catch.  (Reverted.)